aws/asg/variables.tf

```diff
@@ -46,7 +46,10 @@ variable "tags" {
 variable "target_group_arns" {
   type = list(any)
 }
+variable "block_devices" {
+  type = list(any)
```

Comment:
do we set this everywhere? should we add a default?

Reply:
I didn't set a default because I directly pass it from the final main.tf so we don't have to define the actual block device config for every asg group separately but just in 1 place.